src/Content/MediaFactory.php

```diff
@@ -31,29 +33,33 @@ class MediaFactory
 
     /** @var Collection */
     private $mediaTypes;
+    private $areas;
 
-    public function __construct(Config $config, MediaRepository $mediaRepository, TokenStorageInterface $tokenStorage)
+    public function __construct(Config $config, Areas $areas, MediaRepository $mediaRepository, TokenStorageInterface $tokenStorage)
```

Comment:
what is `Areas` class for?

Reply:
Currently there's `config`, `files` and `themes`. It allows uploading / reading and listing files in those areas. Right now locations are fixed, but we might want to make them configurable.

Reply:
Can we think of a better name for it? `FilesLocations` maybe?